Sources/SwiftProtobuf/ExtensionFields.swift

```diff
@@ -46,7 +47,8 @@ extension AnyExtensionField {
 ///
 /// The regular ExtensionField type exposes the value directly.
 ///
-public protocol ExtensionField: AnyExtensionField, Hashable, Sendable {
+@preconcurrency
+public protocol ExtensionField: AnyExtensionField, Hashable {
```

Comment:
Can you also provide a little insight into why this one dropped `Sendable` (others didn't) and why some things that we hadn't tagged as `Sendable` needed the `@preconcurrency`?

Reply:
> why this one dropped `Sendable`

`ExtensionField` conforms to `AnyExtensionField`, which already conforms to `Sendable`, making `ExtensionField` also conform to it transitively. I removed the explicit conformance just to tidy things up - it can be added again if you prefer.

> why some things that we hadn't tagged as `Sendable` needed the `@preconcurrency`

When a public protocol `A` conforms to another `Sendable` protocol `B`, protocol `A` inherits the `Sendable` conformance, meaning the `@preconcurrency` annotation has to be bubbled up.

Reply:
Ack, thanks.